subsys/nrf_rpc/nrf_rpc_uart.c

```diff
@@ -56,11 +65,27 @@ static void work_handler(struct k_work *work)
 	uint8_t *data;
 	size_t len;
 	int ret;
+	uint16_t crc_received = 0;
+	uint16_t crc_calculated = 0;
+
 	while (!ring_buf_is_empty(&uart_tr->rx_ringbuf)) {
 		len = ring_buf_get_claim(&uart_tr->rx_ringbuf, &data, NRF_RPC_MAX_FRAME_SIZE);
 		for (size_t i = 0; i < len; i++) {
 			if (hdlc_input_byte(uart_tr, data[i]) == 0) {
+
 				if (uart_tr->hdlc_state == hdlc_state_frame_found) {
+					crc_received = sys_le16_to_cpu(uart_tr->frame_buffer +
```

Comment:
Let's wait a little bit. Now it isn't correct. The pointer is rotated instead of value.
I apologize for the confusing discussion. :)